recipes/rnabob/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+make clean
+make
+make install HOME=$PREFIX
```

Comment:
Yeah, this looks correct. Can you verify that the binary is ending up in $HOME/bin/ or resp. $PREFIX/bin?